.travis.yml

```diff
@@ -1,16 +1,20 @@
 language: ruby
+perl:
```

Comment:
you can lint your travis.yml by using their command line client gem (packaged in dlre)